cmd/inspect/malware.go

```diff
@@ -80,6 +81,18 @@ func executeMalwareAnalysis() error {
 
 	packageVersion := purl.PackageVersion()
 
+	// If package version is empty or latest replace it with actual literal latest version
+	// Reference: https://github.com/safedep/vet/issues/446
+	if packageVersion.GetVersion() == "" || packageVersion.GetVersion() == "latest" {
```

Comment:
Looks good. Should we also add the same logic to

https://github.com/safedep/vet/blob/main/pkg/readers/purl_reader.go